azure-pipelines-test.yml

```diff
@@ -46,8 +46,10 @@ extends:
     # Application build arguments and config map values as key value pairs.
     # Does not contain all buildArguments, the rest located in kultus-pipelines/kultus-admin-ui-testing.yml and kultus-admin-ui-common.yml
     # The values here will override the values defined in the kultus-pipelines repository.
-    buildArgs:
-      VITE_APP_LINKEDEVENTS_API_URI: 'https://linkedevents.api.test.hel.ninja/v1'
-      VITE_APP_CMS_URI: 'https://kultus.app-staging.hkih.hion.dev/graphql'
+    # buildArgs:
+    #   VITE_APP_API_URI: 'https://kultus.api.test.hel.ninja/graphql'
+    #   VITE_APP_API_REPORT_URI: 'https://kultus.api.test.hel.ninja/reports'
+    #   VITE_APP_LINKEDEVENTS_API_URI: https://linkedevents.api.test.hel.ninja/v1
+    #   VITE_APP_CMS_URI: 'https://kultus.app-staging.hkih.hion.dev/graphql'
```

Comment:
If none of the build args are needed anymore in this file, then they could just be removed rather than left as comments to make the file cleaner, or reduced to a single example.